### ISSUE TITLE
Allow installing XeLaTeX via apt-get

### ISSUE THREAD

**Issue by AtivJoshi** [aws/studio-lab-examples#107]
**Is your feature request related to a problem? Please describe.**
Unable to use the PGF image format in matplotlib as it requires the installation of XeLaTeX.

**Describe the solution you'd like**
Allow the installation of XeLaTeX, preferably via apt or apt-get since it cannot be installed using pip or conda

**Describe alternatives you've considered**
Tried installing texlive-core package via conda, but it does not include XeLaTeX.

**Comment by icoxfog417** [comment]
I understand XeLaTeX is a kind of TeX that can include font and can handle Unicode. We do not provide apt install so that to install XeLateX seems to be difficult. Do you try to convert Jupyter Notebook to PDF?

https://nbconvert.readthedocs.io/en/latest/install.html

**Comment by AtivJoshi** [comment]
This also has the same issue. The nbconvert docs asks you to install tex separately. There is no way to install XeLaTeX via pip or conda.

**Comment by icoxfog417** [comment]
I found [tectonic](https://anaconda.org/conda-forge/) is one of the alternative of XeLaTeX. There was not many document about it so that I'm not clear it enables convert to PDF.

https://github.com/jupyter/nbconvert/issues/808

**Comment by AtivJoshi** [comment]
Thanks for the suggestion, it looks lie a great alternative. Although tectonic is dependent on xelatex, installing tectonic does not install xelatex. And matplotlib does not support tectonic yet. I guess the only option is to wait!!

**Comment by icoxfog417** [comment]
The TeX families are complex and not easy to install. Do you write article on Studio Lab? We are interested in the use case to output PDF on Studio Lab. If it is frequent needs. we will consider to install XeLaTeX be default.

**Comment by AtivJoshi** [comment]
No, I need to generate some plots using matplotlib with proper fonts, which is to be included in a latex article.

**Comment by MicheleMonclova** [comment]
Thanks - I will note this as an enhancement request. Can you tell me if this is for school, a class, or a type of project?

**Comment by AtivJoshi** [comment]
This was a college project.

**Comment by icoxfog417** [comment]
I found that export to PDF causes the error because `XeLaTeX` is required. 

![image](https://user-images.githubusercontent.com/544269/175802500-63441f98-af91-44f1-b062-6ef88d0aaa27.png)

![image](https://user-images.githubusercontent.com/544269/175802465-ce8638d8-d0a3-4348-9b37-2af3bf9d3a55.png)
